multimodal/tests/unittests/predictor/test_predictor.py

```diff
@@ -679,3 +681,104 @@ def test_image_bytearray():
     npt.assert_array_equal(
         [prediction_prob_1, prediction_prob_2, prediction_prob_3, prediction_prob_4], [prediction_prob_1] * 4
     )
+
+
+def test_dump_timm_image():
```

Comment:
Can we separate these tests to another file? The reason is that `test_predictor.py` is growing to be too huge and too slow. 

We can consider to add it under the following name

```shell
unittests/predictor/test_predictor_dump_third_party.py
```

Reply:
Good idea. Done!